distribution/helm/vector-aggregator/values.yaml

```diff
@@ -191,6 +193,8 @@ vectorSource:
     # option: "value"
   # Raw TOML config to embed at the vector source (deprecated).
   rawConfig: null
+  # Specific node port to bind if `service.type` is set to `NodePort`.
```

Comment:
```suggestion
  # Specific node port to bind (useful if `service.type` is set to `NodePort`).
```

Reply:
Also, I'd prefer if the `rawConfig` remained the last field. Could we move the `nodePort` above it?